cookbooks/RealtimeChat.py

```
@@ -88,32 +89,45 @@ async def record_audio() -> AsyncGenerator[MediaData]:
                 frames_per_buffer=CHUNK_SIZE,
             )
 
+            from uuid import uuid4
+            message_id = uuid4()
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)